app/uk/gov/hmrc/pushpullnotificationsgateway/services/CallbackValidator.scala

```diff
@@ -41,17 +40,14 @@ class CallbackValidator @Inject()(outboundProxyConnector: OutboundProxyConnector
         CallbackValidationResult(successful = false, Some("Invalid callback URL. Check the information you have provided is correct."))
       }
     } recover {
-      case e: JsValidationException =>
-        Logger.warn(s"Attempted validation of URL ${callbackValidation.callbackUrl} failed with error ${e.getMessage}")
-        CallbackValidationResult(successful = false, Some("Invalid callback URL. Check the information you have provided is correct."))
       case httpException: HttpException =>
         Logger.warn(failedRequestLogMessage(httpException.responseCode))
         CallbackValidationResult(successful = false, Some("Invalid callback URL. Check the information you have provided is correct."))
       case upstreamErrorResponse: UpstreamErrorResponse =>
```

Comment:
I've kept the http errors separate because they have a specific error message for the logs, but maybe we can just leave the non fatal case...